front/app/services/permissions/rules/routePermissions.ts

```diff
@@ -22,22 +23,27 @@ export const MODERATOR_ROUTES = [
   '/admin/insights',
 ];
 
-export const isModeratorRoute = (item: IRouteItem) => {
-  return MODERATOR_ROUTES.includes(item.path);
+export const isModeratorIndexRoute = (item: IRouteItem) => {
+  return MODERATOR_INDEX_ROUTES.includes(item.path);
 };
 
 export const isModeratedProjectRoute = (
   item: IRouteItem,
   user: IUser | null
 ) => {
-  const idRegexp = /^\/admin\/projects\/([a-z0-9-]+)\//;
-  const matches = idRegexp.exec(item.path);
+  const { pathname: pathNameWithoutLocale } = removeLocale(item.path);
+  const idRegexp = /^\/admin\/projects\/([a-z0-9-]+)\/?/;
+  const matches = pathNameWithoutLocale
+    ? idRegexp.exec(pathNameWithoutLocale)
+    : false;
   const pathProjectId = matches && matches[1];
-  return (pathProjectId && isProjectModerator(user, pathProjectId)) || false;
+  return pathProjectId ? isProjectModerator(user, pathProjectId) : false;
```

Comment:
Replaced item.path with pathNameWithoutLocale and made some type check changes accordingly.